infrastructure/cdn-in-a-box/ort/traffic_ops_ort/__init__.py

```diff
@@ -472,6 +472,14 @@ def main() -> int:
 	                    help="Sets the timeout in milliseconds for requests made to Traffic Ops.",
 	                    type=int,
 	                    default=None)
+	parser.add_argument("--via_string_release",
```

Comment:
Not that you need to bother updating ORT.py at all (bless you), but in most other places you call it `--via-string-release` - which I think is better, honestly - but here you have underscores instead of dashes.

Reply:
👍 will do. Yea I used CIAB as the test environment for this, so to be able to pass the option around I had to add it everywhere, including ort.py :)